packages/camera/src/i18n/resources/en.js

```diff
@@ -0,0 +1,70 @@
+const en = {
+  translation: {
+    camera: {
+      permissionDenied: 'Impossible d\'accéder à la caméra : permission refusée.',
```

Comment:
It's in french instead of english and the same for fr.js it is in english instead of french